dartagnan/src/main/java/com/dat3m/dartagnan/parsers/program/ParserLlvm.java

```diff
@@ -20,7 +20,7 @@ public Program parse(CharStream charStream) {
 
         LLVMIRParser parser = new LLVMIRParser(tokenStream);
         parser.setErrorHandler(new BailErrorStrategy());
-        ProgramBuilder pb = new ProgramBuilder(SourceLanguage.LITMUS);
+        ProgramBuilder pb = ProgramBuilder.forLanguage(SourceLanguage.LITMUS);
```

Comment:
Shouldn't it be `SourceLanguage.BOOGIE`? If so, the fact that this was there before and did not cause problems suggests that this is completely useless.

Reply:
This is a snippet of the `LLVMParser` which is not used right now. It should neither be Boogie nor Litmus but `LLVM` instead.